cmd/internal/serverutil/server.go

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
```

Comment:
Please name this file `main.go` or `serverutil.go`

Reply:
Renamed to main.go.